### DISCUSSION TITLE
Addition of CPEs

### DISCUSSION THREAD

**Discussion by Rohit-Kuinkel** [DependencyTrack/dependency-track#2961] [Q&A]
How can CPEs be added automatically for existing components (for which no cpe is yet known)?

**Comment by syalioune** [comment]
Hello @Rohit-Kuinkel,
It is not possible. See [https://owasp.org/blog/2022/09/13/sbom-forum-recommends-improvements-to-nvd.html](sbom-forum-recommends-improvements-to-nvd) for context.
The current approaching solution will be to enable `CPE fuzzy matching` in Dependency Track settings at the risk of increasing false positive. It'll try to fuzzy match the component's name/group with the updated CPE database.
